HoLC/Widgets/TextEditor.cpp

```diff
@@ -356,6 +356,17 @@ void TextEditor::keyPressEvent(QKeyEvent *pEvent)
     }
 }
 
+void TextEditor::wheelEvent(QWheelEvent* event)
+{
+   if ((event->modifiers() == Qt::ControlModifier) && (event->delta() > 0))
+       zoomIn(2);
```

Comment:
zoomIn and Out does not appear to exist in Qt4, I suggest adding an ifdef around the code and do nothing if the qt version is too low. Not sure exectly when these functions were introduced.

Unless it can be easily implemented in some other way for Qt4.